shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumElectCommand.java

```diff
@@ -69,6 +71,16 @@ public int run(CommandLine cl) throws IOException {
       final int TIMEOUT_3MIN = 3 * 60 * 1000; // in milliseconds
       CommonUtils.waitFor("Waiting for election to finalize", () -> {
         try {
+          GetTransferLeaderMessagePResponse transferMsg = jmClient.getTransferLeaderMessage();
```

Comment:
In `RaftJournalSystem` the exceptions are added to a list and then thrown. Therefore, if exceptions are encountered in `RaftJournalSystem` this call will never be executed.
Either we add them to the list without throwing exceptions in `RaftJournalSystem` and check them here, or we transform this `getTransferLeaderMessage` call into its own shell command.